docs/business.yaml

```diff
@@ -105,6 +105,49 @@ paths:
               schema:
                 type: object
                 properties: {}
+              examples:
+                response-consent-continuation-out:
+                  summary: Consent continuation out filing response
+                  value:
+                    filing:
+                      business:
+                        legalName: 0882848 B.C. LTD.
+                        legalType: BEN
+                        identifier: BC0882848
+                        foundingDate: '2024-07-08T15:34:57.844764+00:00'
+                      consentContinuationOut:
+                        foreignJurisdiction:
+                          country: CA
+                          region: SK
+                        courtOrder:
+                          fileNumber: '12345'
+                          effectOfOrder: planOfArrangement
+                      header:
+                        affectedFilings: []
+                        availableOnPaperOnly: false
+                        certifiedBy: "string"
+                        colinIds: []
+                        comments: []
+                        date: '2024-07-18'
+                        deletionLocked: false
+                        documentOptionalEmail: "user@example.com"
+                        effectiveDate: '2024-07-18T07:00:42.552434+00:00'
+                        email: "user@example.com"
+                        filingId: 150056
+                        inColinOnly: false
+                        isCorrected: false
+                        isCorrectionPending: false
+                        isPaymentActionRequired: true
+                        name: consentContinuationOut
+                        paymentAccount: '3445'
+                        paymentStatusCode: 'CREATED'
+                        paymentToken: '38416'
+                        status: 'PENDING'
+                        submitter: 'api-key-account-3445-sandbox'
+        '400':
```

Comment:
Only cover 2 error responses now.
Plan to update with additional error responses for other potential scenarios.

Reply:
as discussed with Argus, maybe we want to add those scenarios covered in the Postman collections.

Reply:
In that case, maybe we can consider putting precise error messages for the bad calls?

Reply:
For sure, I'm thinking of expanding the corresponding example case under the corresponding error responses schema, something like this.
<img width="968" alt="image" src="https://github.com/user-attachments/assets/a1f4da1e-b01c-47ae-958d-c3f1164c66b9">
What do you think of this?

Reply:
I'd say, put them directly under each paths, like different request examples. Since we may have different response messages